pkg/api/server/register_manifest4.go

```diff
@@ -0,0 +1,180 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/containers/podman/v3/pkg/api/handlers/libpod"
+	"github.com/gorilla/mux"
+)
+
+func (s *APIServer) registerManifest4Handlers(r *mux.Router) error {
+	// swagger:operation POST /libpod/manifests/create manifests ManifestCreateLibpod
+	// ---
+	// summary: Create
+	// description: |
+	//   Create a manifest list
+	//
+	//   As of v4.0.0
+	// produces:
+	// - application/json
+	// parameters:
+	// - in: query
+	//   name: name
+	//   type: string
+	//   description: manifest list or index name to create
+	//   required: true
+	// - in: query
+	//   name: image
```

Comment:
Should this be an array?  `podman manifest create` allows for specifying multiple images, so that may come in handy.

Reply:
@vrothberg I'll code the handler such that `image` can be repeated. I updated the swagger to reflect that.